Extension/src/LanguageServer/client.ts

```diff
@@ -868,6 +877,94 @@ export class DefaultClient implements Client {
         return workspaceFolder ? workspaceFolder.name : "untitled";
     }
 
+    public updateClientConfigurations(): void {
+        clients.forEach(client => {
+            if (client instanceof DefaultClient) {
+                const defaultClient: DefaultClient = <DefaultClient>client;
+                defaultClient.configuration.CompilerDefaults = compilerDefaults;
+                defaultClient.configuration.handleConfigurationChange();
+            }
+        });
+    }
+
+    public set defaultCompiler(value: any) {
```

Comment:
This function seems like it shouldn't exist.  It's the same as the setter in settings.ts.